modules/support/src/lib.rs

```diff
@@ -360,6 +370,32 @@ pub trait EVMBridge<AccountId, Balance> {
 	fn set_origin(origin: AccountId);
 }
 
+#[cfg(feature = "std")]
+impl<AccountId, Balance: Default> EVMBridge<AccountId, Balance> for () {
+	fn name(_context: InvokeContext) -> Result<Vec<u8>, DispatchError> {
+		Ok(Default::default())
```

Comment:
I think the returns for `()` impl should be `Err` instead of `Ok`, to indicate there is no bridge.